Readme.md

```diff
@@ -0,0 +1,308 @@
+    ```
+    git tag vA.B.C 
+    git push origin --tags
+    ```
+3. This will automatically trigger the `update-staging-version.yml` workflow inside your repository. 
+
+This process sends the new tag to the [graasp-deploy](https://github.com/graasp/graasp-deploy) repository and creates a new `YYYYMMddhhmm-staging-versions.json` file including the new version. 
+
+For more information on the appropriate tag syntax, see [???]. 
+
+### Deploy a new stack to staging environment
+The Continuous Delivery workflow (`cdelivery-trigger.yml`) is responsible of deploying a new stack of all the ecosystem versions to staging environment. It can be manually run from the Actions tab of the 
+[graasp-deploy](https://github.com/graasp/graasp-deploy) repository. Being a member of the **staging** team is required to perform these steps.
+
+![Action tab](assets/images/learn-github-actions-repository.png)
+
+Whenever you want to deploy **the latest** staging stack available inside the [staging-versions](https://github.com/graasp/graasp-deploy/staging-versions) folder, you should perform the following steps:
+
+1. In the left sidebar, click the **Deploy to staging environment** workflow.
+
+    ![Action select staging workflow](assets/images/actions-select-deploy-staging-workflow.png)
+
+2. Above the list of workflow runs, select Run workflow.
+
+    ![Action dispatch](assets/images/actions-workflow-dispatch.png)
+
+3. (Optional) Use the input text field to specify a `YYYYMMddhhmm-staging-versions.json` file in case you want to deploy an specific stack. This file must be available inside the [staging-versions](https://github.com/graasp/graasp-deploy/staging-versions) folder. 
+
+    ![Workflow dropdown](assets/images/actions-dropdown-select-staging-stack.png)
+
+4. Finally, click on **Run workflow** and wait until the process is finished. 
+
+This process automatically triggers all the `cdelivery-*.yml` workflows present in the repositories specified inside the `YYYYMMddhhmm-staging-versions.json` file. 
+After the workflow finishes the stack (represented by the `YYYYMMddhhmm-staging-versions.json` file) will be promoted to release:
+- It will be included in the `release-versions` folder
+- It will be copied to the `deployed/current-staging-versions.json` file. 
+
+
+### Deploy a new stack to production environment
+The Continuous Deployment workflow (`cdeployment-trigger.yml`) is responsible of deploying a new stack of all the ecosystem versions to production environment. It can be manually run from the Actions tab of the 
+[graasp-deploy](https://github.com/graasp/graasp-deploy) repository. Being a member of the **production** team is required to perform these steps.
+
+![Action tab](assets/images/learn-github-actions-repository.png)
+
+Whenever you want to deploy **the latest** release stack available inside the [release-versions](https://github.com/graasp/graasp-deploy/release-versions) folder, you should perform the following steps:
```

Comment:
RFC: what is meant by **latest** here? The file with the most recent `YYYYMMddhhmm` timestamp in its file name, the file that was last updated in the repo or the last stack that was deployed to staging?

In other words, if someone last used a custom `YYYYMMddhhmm-staging-versions.json` file name (which is not the most up-to-date) to deploy to staging, would running the action deploy the specific version that is now on staging, or the most up-to-date in any case?

I think you can also clarify in the readme directly 👍

Reply:
**Latest** refers to the file with the most recent **YYYYMMddhhmm** timestamp in its file name inside the **release-versions** folder. It could also be performed based on the latest updated file but this is the approach I decided to use, let me know what you think. 

Whenever you perform the action to deploy to an environment, (it does not matter if custom or default file) it deploys ONLY the versions contained inside that file. Specifically it compares the currently deployed versions (inside /deployed directory) with the versions in the file and deploys those which are different